python/tank/platform/bundle.py

```diff
@@ -461,6 +481,27 @@ def get_project_cache_location(self, project_id):
 
         return self.__cache_location[project_id]
 
+    def get_site_cache_location(self):
```

Comment:
Please make part of the property code. We don't need both a `site_cache_location` and a `get_site_cache_location` method.